parserator_web/templates/parserator_web/index.html

```diff
@@ -36,7 +35,82 @@ <h4>Parsing results</h4>
   </div>
 </div>
 {% endblock %}
-
 {% block extra_js %}
-  <script src="{% static 'js/index.js' %}"></script>
-{% endblock %}
+<script>
```

Comment:
the javascript code should be placed in `js/index.js` and not added inline to the template. can you move this code over?